templates/template-gitignore

```diff
@@ -0,0 +1,59 @@
+### Node ###
```

Comment:
Could we not have this stuff in just one spot? Why do we have it in constants and also here?